mantidproject/mantid
# Changed name and return type of isDistribution setter in matrix workspace

Change name of isDistribution setter to setDistribution on matrix workspace, and removed return type for the setter.

**To test:**

Check the refactoring is sensible and address the problems listed in #15827. Verify all tests pass.

Fixes #15827 .

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
